### ISSUE TITLE
Simplify access to multiple audio channels on pluzz

### ISSUE THREAD

**Issue by sleveque** [rg3/youtube-dl#8495]
Hello

I might be missing something, but I can't find a straightforward way to get the OST version of a video on pluzz.fr, which of course defaults to the French version.

I'm using the latest youtube-dl version packaged for Debian.
```
$ youtube-dl --version
2015.11.27.1
$ youtube-dl -U
It looks like you installed youtube-dl with a package manager, pip, setup.py or a tarball. Please use that to update.
```

Here's the verbose output when I try to obtain the video:
```
$ youtube-dl -vF http://pluzz.francetv.fr/videos/trapped_saison1_ep1_,135088810.html
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-vF', u'http://pluzz.francetv.fr/videos/trapped_saison1_ep1_,135088810.html']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.11.27.1
[debug] Python version 2.7.9 - Linux-2.6.32-042stab111.12-x86_64-with-debian-8.3
[debug] exe versions: avconv 11.4-6, avprobe 11.4-6, ffmpeg 0.8.17-6, rtmpdump 2.4
[debug] Proxy map: {}
[pluzz.francetv.fr] trapped_saison1_ep1_,135088810: Downloading webpage
[pluzz.francetv.fr] 135088810: Downloading video JSON
[pluzz.francetv.fr] 135088810: Downloading geo restriction info
[pluzz.francetv.fr] 135088810: Downloading m3u8 information
[pluzz.francetv.fr] 135088810: Downloading m3u8 information
[info] Available formats for 135088810:
format code                            extension  resolution note
hls_v5_os-meta                         mp4        multiple   Quality selection URL
m3u8-download-meta                     mp4        multiple   Quality selection URL
hls_v5_os-Version Multilingue (AD/VO)  mp4        256x144     180k , avc1,  mp4a
m3u8-download-180                      mp4        256x144     180k , avc1,  mp4a
hls_v5_os-189-0                        mp4        unknown     189k , mp4a
hls_v5_os-189-1                        mp4        unknown     189k , mp4a
hls_v5_os-303                          mp4        320x180     303k , avc1,  mp4a
m3u8-download-303                      mp4        320x180     303k , avc1,  mp4a
hls_v5_os-575                          mp4        512x288     575k , avc1,  mp4a
m3u8-download-575                      mp4        512x288     575k , avc1,  mp4a
hls_v5_os-831                          mp4        704x396     831k , avc1,  mp4a
m3u8-download-831                      mp4        704x396     831k , avc1,  mp4a
hls_v5_os-1466                         mp4        1024x576   1466k , avc1,  mp4a (best)
```

As you can see from the previous output, it seems there's something about choosing another language than French, but definitely not for the best video quality setting.

Subs are there, no problem with that:
```
$ youtube-dl --list-subs http://pluzz.francetv.fr/videos/trapped_saison1_ep1_,135088810.html
[pluzz.francetv.fr] trapped_saison1_ep1_,135088810: Downloading webpage
[pluzz.francetv.fr] 135088810: Downloading video JSON
[pluzz.francetv.fr] 135088810: Downloading geo restriction info
[pluzz.francetv.fr] 135088810: Downloading m3u8 information
[pluzz.francetv.fr] 135088810: Downloading m3u8 information
Available subtitles for 135088810:
Language formats
fr       vtt, ttml, smi
```

The OST is definitely available when streaming (click on the headphones icon in the Flash player to select it, as it is not the default). I have a limited knowledge of how all this works together, but would definitely like to learn how to analyze this, so please do not hesitate if you need technical inputs from me.

The bottom line is that I'd like to understand whether it is a limitation of youtube-dl or if a user action is expected because the software can't be aware that multiple audio possibilities are available. If I have to do something, what path should I have followed to get what I want? The default video that gets downloaded only has the French track. I hope there won't be any geo-related issues.

Thanks for your help

**Comment by sleveque** [comment]
Being an occasional user of `youtube-dl` didn't help me, but I finally understood how I should have proceeded. This is actually a basic situation of downloading a video stream, an audio stream, then merging them together.

`youtube-dl -f hls_v5_os-189-0+hls_v5_os-1466 --all-subs http://pluzz.francetv.fr/videos/trapped_saison1_ep1_,135088810.html` was what I was looking for.

I guess rather than indicating an `unknown` resolution for an audio stream, something saying "this is pure audio" would have definitely put me on the right track.

I didn't check how `youtube-dl` gathers all the information displayed with `-F` but I saw my browser accessing this [manifest](http://replayftv-vh.akamaihd.net/z/streaming-adaptatif_france-dom-tom/2016/S06/J1/135088810-56b8f393893ae-,standard1,standard2,standard3,standard4,standard5,qaa,qad,.mp4.csmil/manifest.f4m). The `label` fields are great hints for selecting the audio, so if this is somewhat standard, it would help having these displayed in `-F`.
